R/download_survey.R

```diff
@@ -0,0 +1,19 @@
+#' @title download_survey
+#'
+#' Access fake mortality survey data
+#'
+#' @param path **Run with no arguments to pick a directory interactively**. Path on your computer where the file(s) should be saved to *string*
+#'
+#' @export
+download_survey <- function(path = rstudioapi::selectDirectory()){
```

Comment:
missin packages importfrom - and also need to add these packages to the description file, and roxygenise to produce namespace with appropriate packages

Reply:
These are already in the Collate part of the description file - is there another section they need to be listed in?

Reply:
Not sure what you are saying about the @importFrom. I ran Ctrl+Shift+D to roxygenise and it said documentation completed. Sorry this is new to me.

Reply:
yeah {rstudioapi} and {cli} need to be listed in the description file under imports (so that they are installed with sitrep). 
You are missing @importfrom in your function scripts. You need to add this as described in point 3 https://github.com/appliedepi/introexerciseslangs/pull/2#issuecomment-1233816040

Reply:
Thanks @aspina7 
I have added cli and rstudioapi to imports in DESCRIPTION, added those two as importFrom in each of the download function scripts, and ran devtools document.